extensions/interactions/GraphInput/GraphInput.js

```diff
@@ -583,6 +583,86 @@ oppia.directive('graphViz', function() {
 
 oppia.factory('graphUtilsService', [function() {
   return {
+    GRAPH_ADJACENCY_MODE: {
+      DIRECTED: 'directed',
+      INVERTED: 'inverted',
+      UNDIRECTED: 'undirected'
+    },
+    DFS_STATUS: {
+      VISITED: 'visited',
+      UNVISITED: 'unvisited',
+      STILL_VISITING: 'still visiting'
+    },
+    /**
+     * @param {object} graph A graph object.
```

Comment:
Ah, there's now a style for this (I had to deal with it during linting). Use a hyphen. E.g.:
```
   * @param {string} assumptionsString - typed by the teacher to describe the
   *        assumptions the student is allowed to make.
```